<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AGS Service Publisher</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>513</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>Reports</string>
    </property>
    <addaction name="actionMXD_Data_Sources_Report"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGetInstallInfo"/>
    <addaction name="actionGetExecutablePath"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPublishing">
    <property name="title">
     <string>Publishing</string>
    </property>
    <addaction name="actionPublish_Services"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPublishing"/>
   <addaction name="menuReports"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMXD_Data_Sources_Report">
   <property name="text">
    <string>MXD Data Sources Report</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionGetInstallInfo">
   <property name="text">
    <string>GetInstallInfo</string>
   </property>
  </action>
  <action name="actionGetExecutablePath">
   <property name="text">
    <string>GetExecutablePath</string>
   </property>
  </action>
  <action name="actionPublish_Services">
   <property name="text">
    <string>Publish Services</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
